registry/workload/values-workload.yaml

```diff
@@ -137,7 +137,15 @@ dnation-kubernetes-monitoring:
 loki:
   enabled: false
 promtail:
-  enabled: false
+  config:
+    clients:
+    - url: "https://monitoring.scs.community/loki/api/v1/push"
+      tenant_id: 1
+      basic_auth:
+        username: <loki-username>
+        password: <loki-password>
+  extraArgs:
+  - -client.external-labels=cluster=harbor-cluster
```

Comment:
It looks good to me. Can you also include the documentation, about filling username and password before applying?

Reply:
I added a note into the docs, hope it is enough. See 5bf4284b94a3c92a6817e916b5544383c2647c72